addons/hr_holidays/static/src/js/time_off_calendar.js

```diff
@@ -112,6 +135,27 @@ odoo.define('hr_holidays.dashboard.view_custo', function(require) {
             });
         },
 
+        _onOpenCreate: function() {
+            var self = this;
+            this._super(...arguments);
+            if(this.previousOpen) {
+                this.previousOpen.on('closed', this, function(ev){
+                    // we reload as record can be created or modified (sent, unpublished, ...)
+                    self.reload();
+                });
+            }
+        },
+
+        /**
+        * @private
+        *
+        **/
+        _onOpenEvent: function (event) {
+            var self = this;
+            this._super(...arguments);
+            this.openEventDialog.buttons[2].text = "Delete";
```

Comment:
Maybe instead of tweaking here, maybe we can pass removeButtonText in options while instantiating ```openEventDialog``` and change ```FormViewDialog``` to get button text from options, maybe something like,
```text: options.removeButtonText || _t("Remove")```

Reply:
Thanks @msh-odoo, much appreciated :)